clients/venice-thin-client/src/main/java/com/linkedin/venice/client/schema/RouterBackedSchemaReader.java

```diff
@@ -44,19 +48,27 @@
 public class RouterBackedSchemaReader implements SchemaReader {
   public static final String TYPE_KEY_SCHEMA = "key_schema";
   public static final String TYPE_VALUE_SCHEMA = "value_schema";
-  public static final String TYPE_GET_UPDATE_SCHEMA = "update_schema";
+  public static final String TYPE_VALUE_SCHEMA_ID = "value_schema_ids";
+  public static final String TYPE_UPDATE_SCHEMA = "update_schema";
```

Comment:
I think it will have to change router API.
Also, for these APIs, I think the existing pattern does not contain such `get_xxx` thing, so I'd suggest we just keep it, only change the constant string variable name.

Reply:
`update_schema` name is a little confusing since it can be used to update an existing value schema.
`get_update_schema` is more clear since it indicates it is a read-only op.
If it is hard to change it, I guess we can live with it since it is internal.

Reply:
Yeah, I think it is not easy to update the endpoint name easily...maybe let's just leave it as it is for now...